includes/DataObjects/Domain.php

```diff
@@ -318,17 +318,17 @@ public function setDefaultLanguage(string $defaultLanguage): void
     /**
      * @return string
      */
-    public function getEmailSender(): string
+    public function getEmailReplyAddress(): string
     {
         return $this->emailsender;
     }
 
     /**
-     * @param string $emailSender
+     * @param string $emailReplyAddress
      */
-    public function setEmailSender(string $emailSender): void
+    public function setEmailReplyAddress(string $emailReplyAddress): void
     {
-        $this->emailsender = $emailSender;
+        $this->emailsender = $emailReplyAddress;
```

Comment:
Should the property be renamed as well? Or would that have too many knock-on effects? I'd still argue that it's maybe a good idea, even if we have to rename the database column, to avoid confusion.

Reply:
I was trying to avoid a schema change with this as I wasn't sure if I'd have to throw it in more quickly than it ended up needing to (reordering schema changes is... messy... with the current patch architecture).

Renaming the underlying field here would require a schema change do to the way PDO::FETCH_CLASS works.

I can add a schema change for this though to tidy it up properly.

Reply:
Ah, okay. If it's too much trouble, lmk and I'll happily approve as-is.

Reply:
Patch uploaded :)